### DISCUSSION TITLE
[Feature Request] Add new user from user relation field HUD

### DISCUSSION THREAD

**Discussion by KatieMFritz** [craftcms/cms#8660] [Ideas]
### Description

When selecting an entry from an entry relations field, there's the option to create a new entry. That option isn't there for users. 

![Screen Shot 2019-04-12 at 10 50 43 AM](https://user-images.githubusercontent.com/7085184/56046261-d96a6800-5d10-11e9-9fbc-c8f2874ecb7e.png)

![Screen Shot 2019-04-12 at 10 44 14 AM](https://user-images.githubusercontent.com/7085184/56046210-be97f380-5d10-11e9-95be-5c5d63552982.png)

The use case where I would want this is to add a contact person to an organization, venue, etc. The person doesn't need to have an active user account immediately (or maybe ever), but we're trying to keep all people in the system as Users instead of entries.

Am I missing a setting that would make this possible? Are there consequences to making this available that I'm missing? If not, is it something you'd consider for an upcoming release?

**Comment by KatieMFritz** [comment]
Hi @brandonkelly! Can you tell me where this might be in the roadmap? If it's not for a long time, I might see if I can commission a plugin.

**Comment by brandonkelly** [comment]
We’ve got some changes to user authorization planned for ~3.3 or 3.4, so it would need to happen alongside that at the earliest, if not later.

**Comment by KatieMFritz** [comment]
Do you think it's theoretically possible for us to develop this functionality as a module in the meantime, or would there be really messy implications?
